.github/workflows/pr-tidying.yml

```diff
@@ -0,0 +1,211 @@
+    # Disabled for now
+    if: always() && github.actor != 'dependabot[bot]' && false
+    needs:
+      - default-assignee
+      - issue-in-title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: step-security/harden-runner@1b05615854632b887b69ae1be8cbefe72d3ae423 # v2.6.0
+        with:
+          disable-sudo: true
+          egress-policy: audit
+      - name: Create comment file
+        run: |
+          COMMENT_FILE=$(mktemp -t comment.md.XXXXX)
+          echo "COMMENT_FILE=$COMMENT_FILE" >> $GITHUB_ENV
+          cat >> $COMMENT_FILE << 'ENDOFCOMMENT'
+          ## PR Helper Bot 🤖
+
+          **Thanks for your pull request!**
+          My job is to check for any missing conventions and automatically tidy things up (if I can).
+          You'll find the results below, along with any suggested actions for you to take.
+
+          <details>
+          <summary>Result key</summary>
+          ✅ = Already passing
+          🛠️ = Fixed (now passing)
+          ⚠️  = Consider fixing
+          </details>
+
+          | Convention                           | Result |
+          |:-------------------------------------|:------:|
+          | PR has assignee (defaults to author) | ${{ env.DEFAULT_ASSIGNEE }} |
+          | PR title ends with `(#issue)`        | ${{ env.ISSUE_IN_TITLE }}   |
+
+          *Pusher: @${{ env.GH_ACTOR }}, Action: `${{ env.GH_ACTION }}`, Workflow: [`${{ env.GH_WORKFLOW }}`](${{ env.GH_SERVER}}/${{ env.GH_REPO }}/actions/runs/${{ env.GH_RUN_ID }})*
+          ENDOFCOMMENT
+        env:
+          DEFAULT_ASSIGNEE: ${{ (needs.default-assignee.outputs.check-passed && '✅' ) || (needs.default-assignee.outputs.fixed && '🛠️') || '⚠️' }}
+          ISSUE_IN_TITLE: ${{ (needs.issue-in-title.outputs.check-passed && '✅' ) || (needs.issue-in-title.outputs.fixed && '🛠️') || '⚠️' }}
+          GH_ACTOR: ${{ github.actor }}
+          GH_ACTION: ${{ github.event_name }}
+          GH_WORKFLOW: ${{ github.workflow }}
+          GH_SERVER: ${{ github.server_url }}
+          GH_REPO: ${{ github.repository }}
+          GH_RUN_ID: ${{ github.run_id }}
+      - name: Output comment body to step summary
+        run: cat $COMMENT_FILE >> $GITHUB_STEP_SUMMARY
+      - name: Write the comment body
+        id: comment-body
+        run: |
+          CONTENT=$(cat $COMMENT_FILE)
+          echo "COMMENT_CONTENT<<ENDOFREPORT" >> $GITHUB_OUTPUT
+          echo "$CONTENT" >> $GITHUB_OUTPUT
+          echo "ENDOFREPORT" >> $GITHUB_OUTPUT
+      - name: Find previous report comment
+        id: find-comment
+        uses: peter-evans/find-comment@a54c31d7fa095754bfef525c0c8e5e5674c4b4b1 # v2.4.0
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: 'PR Helper Bot 🤖'
+      - name: Create or update comment
+        uses: peter-evans/create-or-update-comment@23ff15729ef2fc348714a3bb66d2f655ca9066f2 # v3.1.0
```

Comment:
We've found adding a size label to prs helps people manager their reviewing time more easily - https://github.com/marketplace/actions/pull-request-size-labeler

Reply:
Oh neat! I'll look into pulling that in here – I can definitely see how that would be useful. Thanks for the suggestion!